source/accessible-media/index.html.erb.md

```diff
@@ -19,55 +19,50 @@ contributors:
--   **Transcript**: A text version of a program's audio track,
-    containing narration or dialog plus non-speech information such as
-    speaker-identification cues. Transcripts can be static documents or
-    they can be interactive: e.g., a transcript can be synchronized so
-    that words are highlighted automatically as they are spoken, or
-    words can selected by the user in order to move to that spot in the
+-   [**Captions**](production-captions.html) provide a text version of the video’s dialogue, narration,
+    or other spoken words. They are synchronized with the program audio and
+    displayed within the media player. Include text
+    identifying audible non-speech information, such as music and sound effects.
+-   [**Subtitles**](subtitles.html) are a synchronized textual version of a media’s audio track,
+    translated into another language.
+-   [**Transcripts**](transcript.html) are a text version of a video’s audio track,
+    containing narration, dialog and non-speech information such as
+    speaker-identification cues. Transcripts can be static or
+    interactive documents. Interactive transcripts can be synchronized to the media: words are highlighted automatically as they are spoken, or
+    users can select words to move to that spot in the
     video.
+-   [**Audio descriptions**](production-audio-description.html) are an extra track of narration that conveys
+    information about important visuals, such as body
+    language, changes in scenery and context, charts and
+    diagrams.
 
-## Quality considerations
+## What are good quality captions and audio descriptions?
 
-The quality of the captions and descriptions that are delivered to users
-is of high importance. In the same way that low-quality, incomplete
+Low-quality, incomplete
 audio is unacceptable to hearing viewers, captions that do not
-accurately reflect the audio, or contain spelling and grammatical
+accurately reflect the audio, contain spelling, or grammatical
 errors, are unacceptable to deaf and hard-of-hearing viewers. Similarly,
 audio descriptions that provide misleading or inaccurate information, or
 which provide unnecessary detail (e.g., by giving away clues or
-answers), are of no use to viewers who are blind or visually impaired.
+answers), are of less use to viewers who are blind or visually impaired.
 
 ## Why is this important?
 
 Besides providing crucial communication access for viewers who are deaf, hard of hearing, blind or visually impaired, captions and descriptions have benefits beyond their original target
 audiences:
 
--   Both captions and descriptions can be used to help students and adults with cognitive disabilities engage more directly with media, helping to focus attention or to provide comprehension support.
+-   Students and adults with cognitive disabilities can engage more directly with media when captions and audio descriptions are available as they help to focus attention or provide comprehension support.
```

Comment:
Active instead of passive voice. With the tutorials we always tried to have the subject at the front.